manage/ships/view.php

```diff
@@ -13,14 +13,14 @@
 // check if the ID is valid. 
 $ship = $shipLoader->findOneById($id);
 
-require 'layout/header.php';
+require '../layout/header.php';
 
 ?>
 
     <div class="container">
         <ul class="breadcrumb">
-            <li><a href="/index.php">Home</a></li>
-            <li><a href="/manageShips.php">ManageShips</a></li>
+            <li><a href="/">Home</a></li>
+            <li><a href="manageShips.php">ManageShips</a></li>
```

Comment:
Update this route when you change the name to `index.php`. Then use the full route path: `/manage/ships/index.php`

Reply:
Updated